week-2/implement/cases.js

```diff
@@ -13,3 +13,8 @@
 
 // You will need to come up with an appropriate name for the function
 // Use the string documentation to help you find a solution
+
+function UpperSnakeCase(str){
+    return str.toUpperCase().replace(/ /g, '_');
```

Comment:
do you know what `/ /g` means?

Reply:
Yes, it is a regular expression which means to replace all white spaces with  "-" globally.